openmc/material.py

```diff
@@ -1452,42 +1452,47 @@ def from_xml_element(cls, elem: ET.Element) -> Material:
 
         """
         mat_id = int(elem.get('id'))
-        mat = cls(mat_id)
-        mat.name = elem.get('name')
-
-        if "temperature" in elem.attrib:
-            mat.temperature = float(elem.get("temperature"))
-
-        if 'volume' in elem.attrib:
-            mat.volume = float(elem.get('volume'))
-        mat.depletable = bool(elem.get('depletable'))
-
-        # Get each nuclide
-        for nuclide in elem.findall('nuclide'):
-            name = nuclide.attrib['name']
-            if 'ao' in nuclide.attrib:
-                mat.add_nuclide(name, float(nuclide.attrib['ao']))
-            elif 'wo' in nuclide.attrib:
-                mat.add_nuclide(name, float(nuclide.attrib['wo']), 'wo')
-
-        # Get each S(a,b) table
-        for sab in elem.findall('sab'):
-            fraction = float(sab.get('fraction', 1.0))
-            mat.add_s_alpha_beta(sab.get('name'), fraction)
-
-        # Get total material density
-        density = elem.find('density')
-        units = density.get('units')
-        if units == 'sum':
-            mat.set_density(units)
+        # Add NCrystal material from cfg string
+        if "cfg" in elem.attrib:
+            cfg = elem.get("cfg")
+            mat = Material.from_ncrystal(cfg, material_id=mat_id)
```

Comment:
Rather than having an `else` block below, you can change this to:
```suggestion
            return Material.from_ncrystal(cfg, material_id=mat_id)
```
which means everything that follows can remain unchanged from what it originally was.